docker/build_and_push_go_generate_dependency.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# This is a temporary script, only to be used until we have a better official
```

Comment:
Can you create an issue for this and add a TODO here pointing to the issue? This way, we can track this and backtrack to the appropriate location.

Reply:
Thanks.. Added